### PR TITLE
Better document max TPS tests

### PR DESCRIPTION
**Pull request by bboston7**
Partially addresses #170.

This change adds far more detail to our max TPS documentation with the goal that someone unfamiliar with the missions may be able to run them.

The updated docs also explain the new mixed mode mission, and so should be merged after #164.

### CODE REVIEW

**Review by anupsdf — APPROVED** [review]

File: doc/measuring-transaction-throughput.md
Comment (bboston7) [reply doc/measuring-transaction-throughput.md]: Done!

File: doc/measuring-transaction-throughput.md
Comment (bboston7) [reply doc/measuring-transaction-throughput.md]: I added more details to clear this up

File: doc/measuring-transaction-throughput.md
Comment (bboston7) [reply doc/measuring-transaction-throughput.md]: I added a warning to not run any `perftest` builds in production. I didn't want to just say "dangerous" without expanding on it because I don't want people to think that these images could harm the overall network (they can't). I tried to make it clear that these are dangerous in that they may corrupt your own local node state.

File: doc/measuring-transaction-throughput.md
Comment (bboston7) [reply doc/measuring-transaction-throughput.md]: Removed!

File: doc/measuring-transaction-throughput.md
Comment (bboston7) [reply doc/measuring-transaction-throughput.md]: Removed!

File: doc/measuring-transaction-throughput.md
Comment (bboston7) [reply doc/measuring-transaction-throughput.md]: Done!

**Comment by marta-lokhova** [comment]
We should merge this after merging https://github.com/stellar/supercluster/pull/164

**Comment by marta-lokhova** [comment]
https://github.com/stellar/supercluster/pull/164 and https://github.com/stellar/supercluster/pull/177 are merged now, so we can merge this PR. Looks like there's a merge conflict though.

**Comment by bboston7** [comment]
> Looks like there's a merge conflict though

Resolved!

**Review by marta-lokhova — APPROVED** [review]

**Pull request merged**
